ac/ac-induction/src/Dashboard.jsx

```diff
@@ -0,0 +1,142 @@
+      <div style={{ display: 'flex', flexDirectioon: 'row', margin: '10px' }}>
+        {parts.map(
+          (x, i) =>
+            x > 0 ? (
+              <div
+                key={x.toString() + i.toString()}
+                style={{ flexDirectioon: 'column' }}
+              >
+                <div>{partsNames[i]}</div>
+                <div
+                  className="progress"
+                  style={{ width: x > 0 ? '150px' : '0px' }}
+                >
+                  <div
+                    className="progress-bar progress-bar-striped"
+                    role="progressbar"
+                    aria-valuenow={Math.round(
+                      100 * data[partsNames[i]][0] / (nbInst * x)
+                    )}
+                    aria-valuemin="0"
+                    aria-valuemax="100"
+                    style={{
+                      width:
+                        Math.round(
+                          100 * data[partsNames[i]][0] / (nbInst * x)
+                        ).toString() + '%'
+                    }}
+                  >
+                    {Math.round(
+                      100 * data[partsNames[i]][0] / (nbInst * x)
+                    ).toString() + '%'}
+                  </div>
+                </div>
+              </div>
+            ) : (
+              <div key={x.toString() + i.toString()} />
+            )
+        )}
+      </div>
+      <div>% of the instances that are done with the part: </div>
+      <div style={{ display: 'flex', flexDirectioon: 'row', margin: '10px' }}>
+        {parts.map(
+          (x, i) =>
+            x > 0 ? (
+              <div
+                key={x.toString() + i.toString()}
+                style={{ flexDirectioon: 'column' }}
+              >
+                <div>{partsNames[i]}</div>
+                <div
+                  className="progress"
+                  style={{ width: x > 0 ? '150px' : '0px' }}
+                >
+                  {x > 0 && (
+                    <div
+                      className="progress-bar progress-bar-striped"
+                      role="progressbar"
+                      aria-valuenow={Math.round(
+                        100 * data[partsNames[i]][1] / nbInst
+                      )}
+                      aria-valuemin="0"
+                      aria-valuemax="100"
+                      style={{
+                        width:
+                          Math.round(
+                            100 * data[partsNames[i]][1] / nbInst
+                          ).toString() + '%'
+                      }}
+                    >
+                      {Math.round(
+                        100 * data[partsNames[i]][1] / nbInst
+                      ).toString() + '%'}
+                    </div>
+                  )}
+                </div>
+              </div>
+            ) : (
+              <div key={x.toString() + i.toString()} />
+            )
+        )}
+      </div>
+    </div>
```

Comment:
Your dashboard has two very similar parts, that's what React components are for. Make this into one react component that you use twice with different data and title